### PR TITLE
Update how svg image file is being read

### PR DESCRIPTION
**Pull request by sosharm**
# Related Issue
#8892 

# Description
LoadFromUriAsync() gives a file not found error even if the file exists in WinUI3. Updated to read as string and parse into xml.
There is room for refactoring the SVG xml handling code. 

# Sample Card
1. Existing test: https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.5/Tests/Image.Svg.json

2. `{
  "type": "AdaptiveCard",
  "version": "1.6",
  "backgroundImage": {
    "url": "https://adaptivecards.io/content/Mostly%20Cloudy-Background.jpg"
  },
  "body": [
    {
      "type": "Image",
      "url": "https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/car.svg",
      "size": "large"
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
}`

# How Verified
Verified using the WinUI3 visualizer
1. https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.5/Tests/Image.Svg.json
image

2. 
image

### CODE REVIEW

File: source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp [hunk 1]
```diff
@@ -414,36 +414,45 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
...
+                                        auto success = strongThis->ParseXmlForHeightAndWidth(xmlDoc, strongImageSource);
+                                        if (success)
+                                        {
+                                            // Now that we've parsed the height and width successfully, we can set the image source
+                                            strongThis->SetSvgUriSource(strongImageSource, imageUrl);
+                                        }
+                                    }
                                 }
-                            }
-                            else if (status == winrt::AsyncStatus::Error)
```
Comment (anna-dingler) [inline source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp:440]: Is there a reason why we don't check for this error now?

Comment (sosharm) [reply source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp:440]: I accidentally added it in a previous PR. We are not doing anything with the error.

File: source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp [hunk 2]
```diff
@@ -414,36 +414,45 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
...
-                        if (strongThis && strongImageSource)
+                        if (response.IsSuccessStatusCode())
                         {
-                            if (status == winrt::AsyncStatus::Completed)
-                            {
-                                auto success = strongThis->ParseXmlForHeightAndWidth(operation.GetResults(), strongImageSource);
+                            auto readOperation = response.Content().ReadAsStringAsync();
 
-                                if (success)
+                            readOperation.Completed([weakThis, weakImageSource, imageUrl](auto const& operation, auto status)
```
Comment (beervoley) [inline source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp:432]: this entire chaining of `Completed` handlers is quite bad but Joseph is working on refactoring entire imageRenderer to use coroutines directly.

**Review by beervoley — APPROVED** [review]
Approved. @jwoo-msft will follow up to refactor this.

**Review by jwoo-msft — APPROVED** [review]

**Pull request merged**